<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" idbasedtr="true">
 <author>Jared Gross</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1414</width>
    <height>784</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab_students">
       <attribute name="title">
        <string>Students</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
          <item>
           <widget class="QScrollArea" name="scrollArea_2">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents_2">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>1104</width>
               <height>681</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_students"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Students:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="listWidget_students"/>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_add_student">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Add Student</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_courses">
       <attribute name="title">
        <string>Courses</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>1370</width>
             <height>683</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_courses"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1414</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCourses">
    <property name="title">
     <string>Courses</string>
    </property>
    <addaction name="actionAdd_Course"/>
    <addaction name="actionDelete_Course"/>
   </widget>
   <widget class="QMenu" name="menuStudents">
    <property name="title">
     <string>Students</string>
    </property>
    <addaction name="actionAdd_Student"/>
    <addaction name="actionDelete_Student"/>
   </widget>
   <addaction name="menuCourses"/>
   <addaction name="menuStudents"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd_Course">
   <property name="text">
    <string>Add Course</string>
   </property>
  </action>
  <action name="actionDelete_Course">
   <property name="text">
    <string>Delete Course</string>
   </property>
  </action>
  <action name="actionAdd_Student">
   <property name="text">
    <string>Add Student</string>
   </property>
  </action>
  <action name="actionDelete_Student">
   <property name="text">
    <string>Delete Student</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
